services/github/github_webhook.go

```diff
@@ -368,37 +368,6 @@ func (s *WebhookService) deleteHook(owner, repo string) error {
 	return err
 }
 
-func sameRepos(a *WebhookService, b *WebhookService) bool {
```

Comment:
Why was this deleted? Unusued?

Reply:
Yes, there's nothing using it.

```
$ ag sameRepos
services/github/github_webhook.go
371:func sameRepos(a *WebhookService, b *WebhookService) bool {
```

The only thing that mentions it is the function definition itself. I can't find anything that calls it.

Reply:
> func sameRepos is unused (U1000) go-staticcheck